src/UIKit/UIContentSizeCategory.cs

```diff
@@ -0,0 +1,38 @@
+﻿#if !WATCH
+
+using System;
+using System.Runtime.InteropServices;
+
+using XamCore.ObjCRuntime;
+using XamCore.Foundation;
+
+namespace XamCore.UIKit {
+	static public partial class UIContentSizeCategoryExtensions {
+
+		[DllImport (Constants.UIKitLibrary)]
+		static extern nint /* NSComparisonResult */ UIContentSizeCategoryCompareToCategory (IntPtr /* NSString */ lhs, IntPtr /* NSString */ rhs);
```

Comment:
^ availability attributes are needed on the p/invoke too
the intro tests will look for the symbol and need to report it as missing (in iOS11 but not in earlier versions)

Reply:
Ah, didn't think it was needed since it's private